### DISCUSSION TITLE
Image Indexing with Dall-E and Midjourney

### DISCUSSION THREAD

**Discussion by hra42** [photoprism/photoprism#3889] [Q&A] [answered]
I create a lot of images with ai, like dall-e and midjourney. PhotoPrism isn't recognizing the images and then I have to manully allow them to appear in the search results. Is there any way to make the detection work and allow the photos to be indexed automaticly?

**Comment by lastzero** [comment, marked as answer]
You can disable the filter in Settings or eg add Exif metadata so that it appears as a typical photo if you still want to filter out other non photographic images.

**Reply by hra42** [reply]
Would it be possible to automate EXIF metadata with tools inside photoprism? 

I deactivated the search filter, so I don't have to deal with that.

**Reply by lastzero** [reply]
Exiftool could be used.

**Reply by hra42** [reply]
Thanks for the help, I will look into it!
